tests/_test_base.bash

```diff
@@ -30,6 +30,12 @@ BEGIN { OFS=":"; FS=":"; }
 # This command is used with absolute homedir set and disabled warnings:
 GPGTEST="$SECRETS_GPG_COMMAND --homedir=$TEST_GPG_HOMEDIR --no-permission-warning --batch"
 
+# Platform stuff:
+if [[ "$GITSECRET_DIST" == "windows" ]]; then
+PS_CMD="ps -l -u";
+else
+PS_CMD="ps -wx -U";
```

Comment:
can we indent these to be consistent with the rest of the file?

Reply:
Of course, see below.